src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -2044,6 +2044,8 @@ The webhook will be sent only if the customer has active chats.
 
 Apart from `id`, which is a required parameter, you also need to include **one of the optional** parameters.
 
+Changing the customer `name`, `email`, `avatar`, and `session_fields` while the customer has an active chat will succeed, but the change will apply after the chat has closed.
```

Comment:
I think that this sounds more natural and emphasises what we want:

>The update will apply after the chat is closed in case you are changing the customer while they have an active chat.

Reply:
How about?

> When updating customer data while the customer has an active chat, the update will take effect after the chat has ended.

Reply:
Much better, thx